src/main/java/org/opensearch/security/tools/SecurityAdmin.java

```diff
@@ -275,7 +278,6 @@ public static int execute(final String[] args) throws Exception {
         String type = null;
         boolean retrieve = false;
         String ksAlias = null;
-        String tsAlias = null;
```

Comment:
Why removing this?

Reply:
https://github.com/opensearch-project/security/pull/1638#discussion_r814084702